modules/sfp_shodan.py

```diff
@@ -120,6 +135,18 @@ def handleEvent(self, event):
             evt = SpiderFootEvent("SEARCH_ENGINE_WEB_CONTENT", str(hosts), self.__name__, event)
             self.notifyListeners(evt)
 
+        if eventName == 'WEB_ANALYTICS_ID':
+            network = eventData.split(": ")[0]
+            analytics_id = eventData.split(": ")[1]
+            rec = self.searchHtml(analytics_id)
+
```

Comment:
Perhaps make this upper block a bit safer by catching an exception in case parsing fails?

Reply:
Fixed. Untested. Copy pasta error handling from another module.